### PR TITLE
added demo index pages

### PR DESCRIPTION
**Pull request by taniasultana-dev**

### CODE REVIEW

File: scss/layout/_banner.scss [hunk 1]
```diff
@@ -29,5 +29,31 @@
...
         }
     }
+
+    &__group-btn {
+        @extend .d-flex, .align-items-center, .gap-3, .justify-content-center;
+        a {
+            @extend .no-underline, .inline-block, .btn-base, .rounded-pill;
+            
+        }
+
```
Comment (mollahdev) [inline scss/layout/_banner.scss:39]: remove extra line

File: scss/layout/_banner.scss [hunk 2]
```diff
@@ -29,5 +29,31 @@
...
+        --background-hover-color: var(--blue-color);    
+    }
+
+    &__secondary-btn {
+        --text-color: var(--bs-white);
+        --text-hover-color: var(--bs-white);
+        --background-color: var(--orange-color);
+        --background-hover-color: var(--sky-blue-color);
+    }
+
```
Comment (mollahdev) [inline scss/layout/_banner.scss:56]: remove extra line

File: scss/utilits/_variable.scss [hunk 3]
```diff
@@ -5,7 +5,19 @@ $tablet:992px;
 $tablet-horizontal:1200px;
 $laptop:1400px;
```
Comment (mollahdev) [inline scss/utilits/_variable.scss:7]: remove extra line

**Review by mollahdev — CHANGES_REQUESTED** [review]

File: scss/layout/_banner.scss (hunk 1 above)
Comment (taniasultana-dev) [reply scss/layout/_banner.scss:39]: done

**Comment by taniasultana-dev** [comment]
extra line problem solve

**Pull request merged**